src/debugger/cordovaDebugAdapter.ts

```
@@ -1085,9 +1093,16 @@ export class CordovaDebugAdapter extends ChromeDebugAdapter {
                 this.terminateSession(errMsg);
             });
 
-            return this.doAttach(port, launchUrl, args.address).then(() => {
-                this.attachedDeferred.resolve(void 0);
-            });
+            return this.doAttach(port, launchUrl, args.address)
+                .then(() => {

```

2 comments above are also applicable here
- [ ] move `catch` higher i npromise chain
- [ ] rethrow error instead of just returning it